unzipOne.js

```diff
@@ -0,0 +1,42 @@
+const yauzl = require('yauzl')
+const { basename, join } = require('path')
+const fs = require('fs')
+
+/**
+ * unzip and write file 'fn'  to 'dstDir'
+ * @param {String} srcZip source zipfile
+ * @param {String} fn to unzip
+ * @param {String} dstDir destination dir
+ * @returns {Promise<String>} to output filename if successful, or falsy if the file was not found.
+ */
+function unzipOne (srcZip, fn, dstDir) {
+  const outFile = join(dstDir, fn)
+  return new Promise((resolve, reject) => {
+    yauzl.open(srcZip, { lazyEntries: true },
+      (err, zipfile) => {
+        /* istanbul ignore next */
```

Comment:
Not sure if istanbul is being used for coverage here

Reply:
i use tap which calls istanbul